.gitattributes

```diff
@@ -30,3 +30,11 @@
 
 /.gitattributes export-ignore
 /.gitignore export-ignore
+
+/modules/database/audit-autoloaded-options/readme.txt export-ignore
+modules/database/sqlite/readme.txt export-ignore
+/modules/images/dominant-color/readme.txt export-ignore
+/modules/images/fetchpriority/readme.txt export-ignore
+/modules/images/webp-support/readme.txt export-ignore
+/modules/images/webp-uploads/readme.txt export-ignore
```

Comment:
The requirements state to only add a readme for WebP Uploads as part of this issue. Let's stick to that and ignore all other modules.